src/c++/perf_analyzer/genai-pa/tests/test_cli.py

```diff
@@ -24,14 +24,79 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+import io
+import sys
+
 import pytest
```

Comment:
Should `pytest` be listed as a dependency since it's needed for the testing which we're including in the repo?

@dyastremsky @debermudez

Reply:
Thats a good call.
I think the cleanest way to do this will be to get a container set up, with a virtual env. 
Install whatever is necessary to get genai-pa running, and call pip freeze -r requirements.txt.
I will make a ticket to track this.

Reply:
I suppose that's a broader initiative. I was just talking about adding one dependency (`pytest`) since we're using it for testing and the testing is published to the public repo.

Reply:
`pytest` can also be added to the package dependencies here for our own convenience if needed when running `pip install` here: https://github.com/triton-inference-server/client/blob/394731ddb1e2bd0c8cb9c2c530ce1b60dd91210d/src/c%2B%2B/perf_analyzer/genai-pa/pyproject.toml#L48

Reply:
I thought there was another way. Thanks for calling it out.
@dyastremsky Lets do that with this commit.

Reply:
Added!